subsys/dfu/src/dfu_target_mcuboot.c

```diff
@@ -36,9 +37,10 @@ int dfu_target_mcuboot_init(void)
 	return 0;
 }
 
-int dfu_target_mcuboot_offset(void)
-{
-	return offset;
+int dfu_target_mcuboot_offset(size_t *out)
```

Comment:
This does not match the signature in the declaration in the .h file

Reply:
how? that it does not say `offset`? Does it need to?

A signature is only a parameter-type-list meaning the declaration does not need to match the parameter name.

Reply:
Not by the name, but by the parameter. In `dfu_target_mcuboot.h` it says
`int dfu_target_modem_offset(void);`

Reply:
The same for the `modem` version. Only the signature in the `dfu_target.h` header file has the new parameter type AFAICS

Reply:
True, they were not used that's why I didn't see this. Moved stuff around so they are included for documentation purposes.